lib/python/treadmill_aws/ec2client.py

```diff
@@ -189,6 +206,169 @@ def get_vpc_id_by_tags(ec2_conn, tags):
     return get_vpc_by_tags(ec2_conn, tags)['VpcId']
 
 
+def create_image(
+        ec2_conn, image_name, cloud_init, base_image_id,
+        image_instance_type, image_instance_key, image_instance_role,
+        image_instance_secgroup_ids, image_subnet_id, image_version
+):
+    # pylint: disable=R0915
```

Comment:
now - pylint is write in this case. it is a long function, and it can be easily refactored (no need, as the logic is wrong, so it goes away regardless).